solr/core/src/test/org/apache/solr/handler/TestContainerPlugin.java

```diff
@@ -73,6 +79,29 @@ public void teardown() {
     System.clearProperty("enable.packages");
   }
 
+  public void testZkBehavior() throws Exception {
```

Comment:
This unit test is misplaced, it has nothing to do with this suite. It should be moved to `ZkSolrClientTest`.

Reply:
true